### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by aalysher**
p.s deleted the previous branch with Dockerfile due to merge issues

### CODE REVIEW

**Review by WinterCitizen — DISMISSED** [review]

**Review by UlanDT — APPROVED** [review]

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/roulette:latest
+      - name: Deploy to Server
+        run: |
+          sudo apt-get update && sudo apt-get install -y ansible
+          ansible-playbook -i inventory.ini -u root deploy.yml
```
Comment (WinterCitizen) [inline .github/workflows/cd.yml:31]: @aalysher could you please separate the steps with a newline.

File: Makefile [hunk 2]
```diff
@@ -1,3 +1,7 @@
+install-deploy:
+	poetry config virtualenvs.create false \
+    && poetry install --no-root --without local --no-interaction --no-ansi
```
Comment (WinterCitizen) [inline Makefile:3]: @aalysher only main dependencies should be installed

**Review by WinterCitizen — CHANGES_REQUESTED** [review]

File: .github/workflows/cd.yml (hunk 1 above)
Comment (aalysher) [reply .github/workflows/cd.yml]: done

File: Makefile (hunk 2 above)
Comment (aalysher) [reply Makefile]: done

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,7 @@
+install-deploy:
+	poetry config virtualenvs.create false \
+    && poetry install --no-root --with main --no-interaction --no-ansi
```
Comment (WinterCitizen) [inline Makefile:3]: let it be `--only main`

**Review by WinterCitizen — CHANGES_REQUESTED** [review]

**Review by WinterCitizen — APPROVED** [review]

**Pull request closed without merging**